packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -40,6 +40,8 @@ export interface CompletionContext {
     node?: AstNode
     document: LangiumDocument
     textDocument: TextDocument
+    tokenOffset: number
+    tokenEndOffset: number
     offset: number
     position: Position
```

Comment:
Please add comments to these properties so it's clear how to use them in custom completion code.